cmd/server_test.go

```diff
@@ -620,7 +620,7 @@ tfe-token: my-token
 		"CHECKOUT_STRATEGY":        "branch",
 		"DATA_DIR":                 "/override-path",
 		"DISABLE_APPLY_ALL":        "false",
-		"DEFAULT_TF_VERSION":       "v0.12.0",
+		"DEFAULT_TF_VERSION":       "v0.12.1",
```

Comment:
please don't do a search and replace. It doesn't need to be changed in these tests.

Reply:
Sure, I will just update the necessary changes :) Sorry for the confusion.

Reply:
Hi @lkysow, on a second thought, I think it might be good to keep the version all replaced with v0.12.1.

Keeping both 0.12.0 and 0.12.1 in the codebase might cause some confusion. 

Let me know what is your thought, I can update the PR accordingly.

Reply:
Please just update the 3 files like in this PR: https://github.com/runatlantis/atlantis/pull/400/files

I don't think it causes confusion, those are just tests. There are many version strings throughout the codebase that are needed to be there. In fact on the next version update, there will be less confusion since there will be less instances of 0.12.1 that match a search.